boms/bom/pom.xml

```diff
@@ -108,12 +108,6 @@
         <artifactId>cglib</artifactId>
         <version>3.2.5</version>
       </dependency>
-      <!-- Support for dynamic and type-safe JPA queries -->
```

Comment:
Why was this removed? Does the new spring-boot parent already come with a more recent version of this dependency?

Reply:
I verified that 4.1.4 is used. Seems fine.